parts/2-ngi_pipeline/slides.md

```diff
@@ -1,10 +1,24 @@
-# NGI pipeline
+## TACA - Preprocessing
 
-## Intro to DNA sequencing
-We sequence the source code for life; DNA.
-How do you sequence DNA? What do you do with ACTG?
-Scale of data, automation, backup and clean up, why?
+First and foremost, data needs to go from the sequencer to a machine were the
+preprcessing is done. This is done in local servers because of two reasons.
 
-## TACA
-TACA starts analysis => NGI pipeline
-- configures what analsysis to start
+1. The preprocessing of the data is highly I/O intensive (converting the images
```

Comment:
yup you're right, but I think its easier to explain: "Convert the pictures that the camera of the sequencer takes into A-C-T-G" than the true story... which I wouldn't be able to explain properly even if I wanted, hehe. 

You think its okay, would you rephrase it?